keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ResourceKind.kt

```diff
@@ -0,0 +1,29 @@
+package com.netflix.spinnaker.keel.api
+
+data class ResourceKind(
+  val group: String,
```

Comment:
I find the names to be a bit confusing - what are the valid values for `group` and `kind`?

Reply:
They can be anything, but an example would be group=ec2, kind=cluster

Reply:
I'm happy to change those names if it would be clearer

Reply:
After talking with Luis, he pointed out that this is similar to how k8s is defining resources, so it's probably better to keep it as is for simplicity. Just wondering if we are planning to document it somehow but I guess it all going to be documented once we'll have the open API schemas explaining it?

Reply:
Also, users will not be interacting with those values separately. They'll always use the string form like `ec2/cluster@v1` (see the `toString` of that class)